src/middlewared/middlewared/plugins/smb_/sharesec.py

```diff
@@ -180,21 +168,15 @@ async def _string_to_ae(self, perm_str):
     async def setacl(self, data, db_commit=True):
         """
         Set an ACL on `share_name`. Changes are written to samba's share_info.tdb file.
-        This only impacts SMB sessions. Either ae_who_sid or ae_who_name must be specified
+        This only impacts SMB sessions. ae_who_sid ust be specified
         for each ACL entry in the `share_acl`. If both are specified, then ae_who_sid will be used.
```

Comment:
Typo: ust -> must

Also can lose: "If both are specified, then `ae_who_sid` will be used."